src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1179,6 +1179,29 @@ server.3=zoo3:2888:3888</programlisting>
             </listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term>tcpKeepAlive</term>
+
+            <listitem>
+              <para>(Java system property: <emphasis
+                      role="bold">zookeeper.tcpKeepAlive</emphasisi>)</para>
+
+              <para><emphasis role="bold">New in 3.4.11:</emphasis>
```

Comment:
Please replace 3.4.11 with 3.5.4. 3.4.11 is only applicable for branch-3.4 and 3.5.4 is the version number we use for branch-3.5.